docs/getStarted/local.md

```diff
@@ -27,11 +27,13 @@ Set the following environment variables.
 
  - Unix flavored OSs:
   
+          export KUBERNETES_MASTER=https://172.28.128.4:8443
```

Comment:
This is unnecessary - there was a bug in the client which meant it was required, but that is fixed now & the master address is correctly read from the `~/.kube/config` file.